mackup/applications/cleanshot.cfg

```diff
@@ -3,3 +3,5 @@ name = CleanShot
 
 [configuration_files]
 Library/Preferences/com.getcleanshot.app-setapp.plist
+Library/Preferences/clean
+pl.maketheweb.cleanshotx.plist
```

Comment:
I can't believe it creates such file in the home folder...

Reply:
Holy moley! I'm not sure how that line break got in there. Was fighting with git.  Will fix!